src/js/controllers/imageCtrl-test.js

```diff
@@ -0,0 +1,125 @@
+describe('imagesCtrl', function() {
+    var $rootScope, $controller, imagesCtrl, images, vm, $q;
+
+    beforeEach(module('flickrPOC'));
+
+    beforeEach(module(function($provide){
+        $provide.service('images', function($q){
+            this.loadPics = sinon.stub().returns($q.resolve({message: 'loadPics Working fine'}));
+        });
+
+    }));
+
+    beforeEach(inject(function (_$rootScope_, _$controller_, _images_, _$q_) {
+        $rootScope = _$rootScope_.$new();
+        $controller = _$controller_;
+        images = _images_;
+        $q = _$q_;
+
+        vm = $controller('imagesCtrl');
+
+    }));
+
+    describe('imagesCtrl', function(){
+        beforeEach(function () {
+            images.loadPics.returns($q.resolve(
+                {
+                    contentType: "text/json",
+                    status: "success",
+                    photos: {
+                        photo: [
```

Comment:
Yeah, photo shows up as an array of objects. 
<img width="292" alt="screen shot 2016-01-07 at 11 01 31 am" src="https://cloud.githubusercontent.com/assets/7050734/12175082/3e68e062-b52e-11e5-87fb-7363e21a0511.png">